Updated README

michaelajr opened the pull request:
Removed compatibility with Python SDK - left Java SDK - and linked to compatibility section.

mattsb42-aws commented on README.rst:
```diff
@@ -186,6 +183,11 @@ Compatability with the AWS Encryption SDK
 Encryption SDK.** But not all files encrypted with the AWS Encryption
```
Would you mind updating this statement to specify the Encryption SDK for Java as well? I don't want to send mixed messages, and technically "the AWS Encryption SDK" is the standard, not any specific client.

michaelajr commented:
@mattsb42-aws for your review. I now only reference compatibility the the Java SDK and renamed the compatibly section to be Java-specific. Also updated all references to the "SDK" to be the "Java SDK".

mattsb42-aws commented:
Looks good, thanks!
